### DISCUSSION TITLE
Limit CPU / Threat-Usage

### DISCUSSION THREAD

**Discussion by jkdi** [catchthatrabbit/coreminer#61] [Miner]
hello,
how can i use the --cpu option in coreminer with mining? i need this option...

thx for your replay.

**Comment by rastislavcore** [comment]
I assume you need to specify `-t` or `--threads`.

Or do you need: `--cpu-devices`, `--cp-devices`?

Please, be more specific in your description.

Reference to the code: https://github.com/catchthatrabbit/coreminer/blob/master/coreminer/main.cpp#L253

**Reply by jkdi** [reply]
i use version v19.63. i see, that this option exists in v68! this solve my problem... thx!
